src/auth/constants/auth.constants.ts

```diff
@@ -19,3 +19,14 @@ export const REFRESH_TOKEN_ERROR_CODES = {
   REFRESH_TOKEN_NOT_FOUND: 'REFRESH_TOKEN_NOT_FOUND',
   INVALID_REFRESH_TOKEN: 'INVALID_REFRESH_TOKEN',
 } as const;
+
+export const AUTH_DOMAIN_ERRORS: Record<string, { field: string; reason: string }> = {
+  USERNAME_ALREADY_EXISTS: {
+    field: 'username',
+    reason: 'This username is already in use',
+  },
+  NICKNAME_ALREADY_EXISTS: {
+    field: 'nickname',
+    reason: 'This nickname is already in use',
+  },
```

Comment:
이미 다른곳에서 사용중이라 사용하지 못할때는 `occupied`를 많이 사용하곤 해요. 물론 지금도 좋습니다

```suggestion
  USERNAME_ALREADY_OCCUPIED: {
    field: 'username',
    reason: 'This username is already occupied',
  },
  NICKNAME_ALREADY_OCCUPIED: {
    field: 'nickname',
    reason: 'This nickname is already occupied',
  },
```